Index: Chetand/Seleniumpythonprogramming/Python_loop_Conditions/Python_loop_Session.py
===================================================================
diff --git a/Chetand/Seleniumpythonprogramming/Python_loop_Conditions/Python_loop_Session.py b/Chetand/Seleniumpythonprogramming/Python_loop_Conditions/Python_loop_Session.py
deleted file mode 100644
--- a/Chetand/Seleniumpythonprogramming/Python_loop_Conditions/Python_loop_Session.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,28 +0,0 @@
-print("@"*50)
-
-for i in range(1, 20, 3):
-
-    print(i)
-
-# 2nd Example#
-
-print("@"*50)
-
-for j in range(1, 20):
-
-    print(j)
-
-# 3rd Example#
-
-print("@" * 50)
-
-for k in range(3):
-    print(k)
-
-# 4th Example Get all the even numbers from 1 to 20#
-print("@" * 50)
-for i in range(1, 20):
-    if i % 2 == 0:
-        print(i)
-
-
Index: Sushmita/pythonprogram/If_else condition/authentication.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/authentication.py b/Sushmita/pythonprogram/If_else condition/authentication.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/authentication.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,6 +0,0 @@
-username=input('Enter name :')
-password =input('Enter password :')
-if username == password :
-    print('Authentication is valid')
-else :
-    print('Authentication is not valid')
Index: Neha/PythonVariables/PythonVariables.py
===================================================================
diff --git a/Neha/PythonVariables/PythonVariables.py b/Neha/PythonVariables/PythonVariables.py
deleted file mode 100644
--- a/Neha/PythonVariables/PythonVariables.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,95 +0,0 @@
-var1 = 50
-print(var1)
-print(type(var1))
-print(id(var1))
-
-print("_"*50)
-var2 = "Happy Sunday"
-print(var2)
-print(type(var2))
-print(id(var2))
-
-print("_"*50)
-# multiple variable with same value then their address will be same
-a=50
-b=100
-print(id(a))
-print(id(b))
-
-print("_"*50)
-# assign multiple values to multiple variables at a time
-p, q, r =50, 50, "Smera"
-print("Value of p:", p, id(p))
-print("Value of q:", q, id(q))
-print("value of r:", r, id(r))
-
-print("_"*50)
-# Assign one value to multiple variables
-x= y= z= 100
-print("Value of x:", x, id(x))
-print("Value of y:", y, id(y))
-print("Value of z:", z, id(z))
-
-print("_"*50)
-# plus operator(+)
-a = 10
-b = 30
-c = a+b
-print("Addition-", c)
-
-print("_"*50)
-# Minus operator(-)
-a = 10
-b = 30
-c = a-b
-print("Substraction-", c)
-print("Substraction-", b-a)
-
-print("_"*50)
-# Multiplication operator(*)
-a = 10
-b = 30
-c = a*b
-d = "Smera "
-print("Multiplication-", c)
-print("Multiplication-", d*a)
-
-print("_"*50)
-# Reminder operator(%)
-a = 10
-b = 31
-print("Reminder-", b%a)
-
-print("_"*50)
-# Power operator(**)
-print("Square of 5 is :", 5**2)
-print("Cube of of 5 is :", 5**3)
-
-print("_"*50)
-# Equal to operator(==)
-a = 100
-b = 200
-c = 100
-print("Compare a and b:", a==b)
-print("Compare c and b:", c==b)
-print("Compare a and c:", a==c)
-
-print("_"*50)
-#(a+b)^2 = a^2 + b^2 + 2ab
-a = 2
-b = 3
-LHS = (a+b)**2
-print("LHS is :", LHS)
-RHS = a**2+b**2+2*a*b
-print("RHS is :", RHS)
-
-print("_"*50)
-#(a/b)+(c/d)=(ad+bc)/bd
-a = 8
-b = 4
-c = 7
-d = 5
-LHS = (a/b)+(c/d)
-RHS = (a*d+b*c)//(b*d)
-print("LHS is :", LHS)
-print("RHS is :", RHS)
\ No newline at end of file
Index: Sushmita/pythonprogram/If_else condition/Even_odd.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/Even_odd.py b/Sushmita/pythonprogram/If_else condition/Even_odd.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/Even_odd.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,5 +0,0 @@
-num=int(input('enter number :'))
-if(num%2==0) :
-    print('number is even ')
-else :
-    print('number is odd')
Index: Sushmita/pythonprogram/If_else condition/id_list.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/id_list.py b/Sushmita/pythonprogram/If_else condition/id_list.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/id_list.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,6 +0,0 @@
-List=[1,3,5,6,7]
-ID=int(input('enter the ID :'))
-if ID in List :
-    print('valid Id')
-else :
-    print('Invalid ID')
Index: Sushmita/pythonprogram/If_else condition/square.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/square.py b/Sushmita/pythonprogram/If_else condition/square.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/square.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,7 +0,0 @@
-n1=int(input('enter number: '))
-num=n1**2
-print('square of number is : ' ,num)
-if num%11==0 :
-    print('number is divisible by 11')
-else :
-    print('number is not divisible by 11')
Index: keerthi/python forloop cw/forloop pratice.py
===================================================================
diff --git a/keerthi/python forloop cw/forloop pratice.py b/keerthi/python forloop cw/forloop pratice.py
deleted file mode 100644
--- a/keerthi/python forloop cw/forloop pratice.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,14 +0,0 @@
-# (first val,end value, difference)
-
-for i in range(1,10):
-    print(i)
-
-print("_"*50)
-for i in range(10):
-    print(i)
-
-print("_"*50)
-for i in range:
-    pass
-
-
Index: Sushmita/pythonprogram/If_else condition/Divsible.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/Divsible.py b/Sushmita/pythonprogram/If_else condition/Divsible.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/Divsible.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,7 +0,0 @@
-n1=int(input('enter a number:'))
-
-if n1%3 == 0:
-    print( f"{n1}  is divisible by 3")
-else :
-
-    print(f"{n1} is not divsible by 3")
Index: Sushmita/pythonprogram/If_else condition/Divide_by3_5.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/Divide_by3_5.py b/Sushmita/pythonprogram/If_else condition/Divide_by3_5.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/Divide_by3_5.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,5 +0,0 @@
-num=int(input('enter number :'))
-if num%3==0 and num%5==0 :
-    print('Given number is divisible by 3 and 5')
-else :
-    print('Given number is not divisible by 3 and 5')
Index: Sushmita/pythonprogram/If_else condition/age_limit.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/age_limit.py b/Sushmita/pythonprogram/If_else condition/age_limit.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/age_limit.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,5 +0,0 @@
-age =int(input('Enter your age :'))
-if age>18 :
-    print('Eligible for voting')
-else :
-    print('Not Eligible for voting')
Index: Sushmita/pythonprogram/If_else condition/interview.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/interview.py b/Sushmita/pythonprogram/If_else condition/interview.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/interview.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,10 +0,0 @@
-Round1=input('Enter Round1 result :')
-Round2=input('Enter Round2 result :')
-if Round1 == 'pass' :
-    print("Selected for Round 2")
-    if Round2 =='pass' :
-          print("selected ,Congrualations")
-    else :
-          print("Failed in 2nd round, Better luck next time")
-else :
-    print("Failed in 1st round,Better luck next time")
Index: Sushmita/pythonprogram/If_else condition/Square_cube.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/Square_cube.py b/Sushmita/pythonprogram/If_else condition/Square_cube.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/Square_cube.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,5 +0,0 @@
-num=int(input('Enter number :'))
-if num%2==0 :
-    print("Square :", num**2)
-elif num%3==0 :
-    print('cube :', num**3)
Index: Sushmita/pythonprogram/If_else condition/list.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/list.py b/Sushmita/pythonprogram/If_else condition/list.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/list.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,6 +0,0 @@
-List=[1,2,3,4,7,8]
-num=int(input('enter number :'))
-if num in List :
-    print(f'{num}  is available is list')
-else :
-    print(f'{num} is not available is list')
Index: Subramanyam/ifelsepractice.py
===================================================================
diff --git a/Subramanyam/ifelsepractice.py b/Subramanyam/ifelsepractice.py
deleted file mode 100644
--- a/Subramanyam/ifelsepractice.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,246 +0,0 @@
-n = 50
-
-result ='even' if n%2==0 else 'odd'
-
-print(result)
-
-
-n1= 66
-
-n2 = [59,60,61,62,63]
-
-result1 = "True" if n1 in n2 else "false"
-
-print(result1)
-
-#  Python program to check given number is divided by 3 or not.
-
-a=34
-
-if a%3==0:
-    print("divisible by 3")
-else:
-    print("not divisible by 3")
-
-#  If else program to get all the numbers divided by 3 from 1 to 30.
-c=1
-for c in range(1,31):
-    if c%3==0:
-        print(c,end=" ")
-
-
-print("$"*10)
-# If else program to assign grades as per total marks.
-'''
-marks > 40: Fail
-marks 40 – 50: grade C
-marks 50 – 60: grade B
-marks 60 – 70: grade A
-marks 70 – 80: grade A+
-marks 80 – 90: grade A++
-marks 90 – 100: grade Excellent
-marks > 100: Invalid marks
-'''
-
-marks = 32
-
-if marks>100:
-    print("invalid marks")
-elif 30 <= marks <= 40:
-    print("pass with grade c")
-elif 40 < marks <= 50:
-    print("pass with grade b")
-elif 50<marks<=60:
-    print("pass with grade a")
-elif 60<marks<=70:
-    print("pass with grade a+")
-elif 70<marks<=80:
-    print("pass with grade a++")
-elif 80<marks<=90:
-    print("pass with grade a+++")
-elif 90<marks<=100:
-    print("pass with grade excellent")
-else:
-    print("student failed")
-
-# Python program to check the given number divided by 3 and 5.
-
-n3 = 45
-
-if n3%3==0 and n3%5==0:
-    print("divisible by 3 and 5")
-else:
-    print("not divisible by 3 and 5")
-
-#  Python program to print the square of the number if it is divided by 11.
-'''
-num = input("Enter the number")
-
-if int(num)%11==0:
-    print('divisible by 11: ',(int(num))**2)
-else:
-    print("not divisible")
-'''
-# Python program to check given number is a prime number or not.
-'''
-num2=int(input("enter the number"))
-count=0
-
-for i in range(2,num2):
-    if num2%i==0:
-        count+=1
-
-
-
-if count>0:
-    print("not a prime")
-else:
-
-    print("prime")
-'''
-# Python program to check given number is odd or even.
-'''
-s=int(input("Enter the number: "))
-
-if s%2==0:
-    print("even")
-else:
-    print("Odd")
-'''
-
-# Python program to check a given number is part of the Fibonacci series from 1 to 10.
-'''
-def fabi():
-
-    num1 = int(input("Enter the number: "))
-
-    num2 = int(input("number2: "))
-
-    count =1
-
-    print("The Sequence is :",end=" ")
-    while count<10:
-        print(num1,end=" ")
-        num3=num1+num2
-        num1=num2
-        num2=num3
-        count+=1
-fabi()
-new=[0,1,1,2,3,5,8,13,21,34,]
-h=22
-if h in new:
-    print("It is part series")
-else:
-    print("not a part series")
-'''
-
-# Python program to check authentication with the given username and password.
-'''
-username=input("Enter username: ")
-password=input("Enter password: ")
-
-if username==password:
-    print("valid")
-else:
-    print("invalid")
-
-'''
-# Python program to validate user_id in the list of user_ids.
-'''
-id_lst=[1,2,3,4,5,6,7,8,9]
-id_l=int(input("Enter the ID: "))
-
-if id_l in id_lst:
-    print("Yes is there")
-else:
-    print("No not there")
-'''
-
-# Print a square or cube of the given number
-'''
-num = int(input("number: "))
-if num%2==0:
-    print("square: ",num**2)
-elif num%3==0:
-    print("cube",num**3)
-else:
-    print("other than: ",num**2)
-
-# Python program to describe the interview process.
-r1=input("Enter the  data1: ")
-r2=input("Enter the data2: ")
-
-if r1=="passed":
-    print("congrats your 1st round is clear")
-    if r2=="passed":
-        print("congrats your 2nd round is clear,you are plced")
-    else:
-        print("failed in 2rd round ,please try next time ")
-else:
-    print("failed in 1st round ,please try next time ")
-
-'''
-# Python program to determine whether a given number is available in the list of numbers or not.
-'''
-lst=[56,2,15,1,41,6,23,2,5,26,2,42,9]
-
-m=int(input("enter the data"))
-
-if m in lst:
-    print(f" {m} is present")
-else:
-    print(f" {m} is not present")
-
-# Python program to find the largest number among three numbers.
-
-num1 =int(input("number1: "))
-num2=int(input("number2: "))
-num3=int(input("number3: "))
-
-if num1>num2:
-    if num1>num3:
-        print(f"{num1} is greater")
-    else:
-        print(f"{num3} is greater")
-
-else:
-    if num2>num3:
-        print(f'{num2} is greater')
-    else:
-        print(f'{num3} is greater')
-
-# Python program to check any person eligible to vote or not
-# age > 18+ : eligible
-# age < 18: not eligible
-
-user=int(input("Enter the information: "))
-
-if user>18:
-    print("User is eligible to vote")
-else:
-    print("user not eligible for vote")
-'''
-# Python program to check whether any given number is a palindrome.
-# Input: 121
-# Output: palindrome
-'''
-num = int(input("Enter the number: "))
-rev = 0
-
-while num>0:
-    rem =n%10
-    rev =rev*10+rem
-    num=num//10
-
-if num==rev:
-    print(f"{num} is palinadrome")
-else:
-    print(f"{num} is not palinadrome")
-
-m =121
-h = str(m)
-if m==int(h[::-1]):
-    print(f"{h} is palinadrome2")
-else:
-    print(f"{h} not palinadrome2")
-'''
Index: Sushmita/pythonprogram/If_else condition/grade.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/grade.py b/Sushmita/pythonprogram/If_else condition/grade.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/grade.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,24 +0,0 @@
-marks=int(input("enter student obtained marks:"))
-if marks < 40 :
-    print("Failed")
-elif marks >=40 and marks <=50:
-    print("Grade C")
-elif marks >=50 and marks <=60:
-    print("Grade B")
-elif marks >=60 and marks <=70:
-    print("Grade A")
-elif marks >=70 and marks <=80:
-    print("Grade A+")
-elif marks >=80 and marks <=90:
-    print("Grade A++")
-elif marks >=90 and marks <=100:
-    print("Grade Excellent")
-else :
-    print("Invalid narks")
-
-
-
-
-
-
-
Index: Mahila/pythonprogramming/pythonforloop/forloopexercise.py
===================================================================
diff --git a/Mahila/pythonprogramming/pythonforloop/forloopexercise.py b/Mahila/pythonprogramming/pythonforloop/forloopexercise.py
deleted file mode 100644
--- a/Mahila/pythonprogramming/pythonforloop/forloopexercise.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,4 +0,0 @@
-
-for i in range(1,20):
-    if i%2==0:
-        print(i)
\ No newline at end of file
Index: Sangeetha/Python_Ifelse/If_Else_practice.py
===================================================================
diff --git a/Sangeetha/Python_Ifelse/If_Else_practice.py b/Sangeetha/Python_Ifelse/If_Else_practice.py
deleted file mode 100644
--- a/Sangeetha/Python_Ifelse/If_Else_practice.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,78 +0,0 @@
-##program to check given number is divided by 3 or not.
-
-num1=21
-
-if num1%3==0:
-    print("number is divisible by 3")
-else:
-    print("number is not divisible by 3")
-
-print("_"*40)
-
-##If else program to assign grades as per total marks.
-mark=55
-
-if mark < 40:
-    print("student is failed")
-elif 40 < mark <= 50:
-    print("student is passed in C grade")
-elif 50 < mark <= 60:
-    print("student is passed in B grade")
-elif 60 < mark <= 70:
-    print("student is passed in A grade")
-elif 70 < mark <= 80:
-    print("student is passed in A+ grade")
-elif 80 < mark <= 90:
-    print("student is passed in A++ grade")
-elif 90 < mark <= 100:
-    print("student is passed in Excellent grade")
-else :
-    print ("Invalid marks")
-
-print("_"*40)
-
-### program to check the given number divided by 3 and 5.
-num2=30
-if num2%3 and num2%5 :
-    print ("number is divisible by 3 and 5")
-else:
-    print("number is not divisible by 3 and 5")
-
-print("_"*40)
-
-##Python program to print the square of the number if it is divided by 11.
-
-num3=int(input("Enter a number: "))
-if num3%11 == 0:
-
-    print(num3**2)
-else:
-    print("number is not divisible by 11")
-
-print("_"*40)
-
-##Python program to check given number is odd or even
-
-num4=int(input("Enter the number: "))
-if num4 % 2 == 0:
-    print("number is even")
-else:
-    print("number is odd ")
-
-print("_"*40)
-
-#Python program to check authentication with the given username and password.
-
-name=input("enter name : ")
-password=input("password : ")
-if name==password:
-    print("it is  valid ")
-else :
-    print("it is not valid")
-
-print("_"*40)
-
-###Python program to validate user_id in the list of user_ids.
-
-
-
Index: Sushmita/pythonprogram/If_else condition/Positive_negitive.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/Positive_negitive.py b/Sushmita/pythonprogram/If_else condition/Positive_negitive.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/Positive_negitive.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,12 +0,0 @@
-num=int(input('Enter number :'))
-if num>0 :
-    if num%2==0 :
-
-        print(f"{num} is positove and even ")
-    else:
-        print(f"{num} is positove and odd ")
-else  :
-    if num%2==0 :
-        print(f"{num} is negative and even ")
-    else:
-        print(f"{num} is negative  and odd ")
Index: Sushmita/pythonprogram/If_else condition/largest_number.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/largest_number.py b/Sushmita/pythonprogram/If_else condition/largest_number.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/largest_number.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,10 +0,0 @@
-n1=int(input('enter a n1 number :'))
-n2=int(input('enter a n2 number :'))
-n3=int(input('enter a n3 number :'))
-if n1>n2 and n1>n3 :
-    print(f'{n1} is largest number')
-elif n2>n1 and n2>n3 :
-    print(f'{n2} is largest number')
-elif n3>n1 and n3>n2 :
-    print(f'{n3} is largest number')
-
Index: Swapna1/Elseif Practice program.py
===================================================================
diff --git a/Swapna1/Elseif Practice program.py b/Swapna1/Elseif Practice program.py
deleted file mode 100644
--- a/Swapna1/Elseif Practice program.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,114 +0,0 @@
-'''
-if condition:
- else:
- code
-''' '''
-    
-
-''
-##Python program to check if given number is divisible by 3 or not##
-a=6
-if a%3 == 0:
- print("Number is divisible by 3")
-else:
- print("Number is not divisible")
-
-####
-a=7
-if a%3 == 0:
- print("Number is divisible by 3")
-else:
- print("Number is not divisible")
-'''
-
-###How to get a number##
-''''
-num=int(input("Enter the value"))
-if num%3 == 0:
- print("Number is divisible by 3")
-else:
- print("Number is not divisible")
- '''
-'''
-#####if else program to assign grade as per total mark####
-marks 40 – 50: grade C
-marks 50 – 60: grade B
-marks 60 – 70: grade A
-marks 70 – 80: grade A+
-marks 80 – 90: grade A++
-marks 90 – 100: grade Excellent
-marks > 100: Invalid marks
-
-
-mark=int(input("Enter the mark"))
-if mark<40:
-    print("Fail")
-elif mark>=40 and mark<50:
-    print("Grade C")
-elif mark>=50 and mark<60:
-    print("Grade B")
-elif mark >= 60 and mark < 70:
-    print("Grade A")
-elif mark >= 70 and mark < 80:
-    print("Grade A+")
-elif mark >= 80 and mark < 90:
-    print("Grade A++")
-elif mark >= 90 and mark <= 100:
-    print("Grade Excellent")
-else:
-    print("Invalid Mark")
-'''
-##### Program to check if the number is divisible by 3 and 5#####
-'''
-num=int(input("Enter the number"))
-if num%3==0 and num%5==0:
-    print("Number is divisible by 3 and 5")
-else:
-    print("Number is not divisble by 3 and 5")
-'''
-#####Program to print the square of the number###
-'''
-num=int(input("Enter the number"))
-if num%11==0:
-    print("Square of the number is", num**2)
-else:
-    print("number is not divisible by 11")
-    '''
-####### program to check  given number is odd or even######
-
-'''
-num=int(input("Enter the number"))
-if num%2==0:
-    print("Even number")
-else:
-    print("Odd number")
-'''
-
-#####Check authentication of the user name and password########
-'''
-num1=str(input("Enter the user name:"))
-num2=str(input("Enter the password:"))
-if num1==num2:
-    print("Valid")
-else:
-    print("Invalid")
-    '''
-
-####validate user_id in the list of user_ids###
-'''
-list_a=[4,7,8,9,34,89,23]
-a=int(input("Enter the user id"))
-if a in list_a:
-    print("User id available")
-else:
-    print("User id is not available")
-    '''
-####### program to print a square or cube if the given number is divided by 2 or 3##
-'''
-num = int(input("Enter the number:"))
-if num%2 == 0:
- print("Square of the given number is:",num**2)
-elif num%3 == 0:
- print("Cube", num**3)
- '''
-#######nested If else program to describe the interview process######
Index: Sunita/New Text Document (2).txt
===================================================================
diff --git a/Sunita/New Text Document (2).txt b/Sunita/New Text Document (2).txt
deleted file mode 100644
--- a/Sunita/New Text Document (2).txt	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,1 +0,0 @@
-print("Hello world")
Index: Neha/PythonIfElseConditions/IfElseExamples.py
===================================================================
diff --git a/Neha/PythonIfElseConditions/IfElseExamples.py b/Neha/PythonIfElseConditions/IfElseExamples.py
deleted file mode 100644
--- a/Neha/PythonIfElseConditions/IfElseExamples.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,77 +0,0 @@
-#Python program to check given number is divided by 3 or not.
-
-a = 32
-if a%3 ==0:
-   print("a is divisible by 3")
-else:
-    print("a is not divisible by 3")
-
-#Python program to check whether the given number is negative or not.
-
-a = -45
-if a<0:
-    print("Number is negative")
-else:
-    print("Number is positive")
-
-#Using a python program take input from the user between 1 to 7 and
-# print the day according to the number. 1 for Sunday 2 for Monday so on.
-
-x = int(input("Enter a number: "))
-
-if x == 1:
-    print("Sunday")
-elif x == 2:
-    print("Monday")
-elif x == 3:
-    print("Tuesday")
-elif x == 4:
-    print("Wednesday")
-elif x == 5:
-    print("Thursday")
-elif x == 6:
-    print("Friday")
-elif x == 7:
-    print("Saturday")
-else:
-    print("Invalid number")
-
-#Python program to accept two numbers and mathematical operations from users and
-# perform mathematical operations according to it.
-
-n1 = int(input("Enter n1: "))
-n2 = int(input("Enter n2: "))
-operation = input("operation: ")
-
-if operation == "+":
-    print(n1+n2)
-elif operation == "-":
-    print(n1-n2)
-elif operation == "*":
-    print(n1*n2)
-elif operation == "/":
-    print(n1/n2)
-else:
-    print("Invalid operation")
-
-
-
-
-#Python program to check whether the citizen is a senior citizen or not.
-#An age greater than 60 than the given citizen is a senior citizen.
-
-age = int(input("Enter the age of citizen:"))
-if age>60:
-    print("Citizen is senior citizen")
-else:
-    print("Citizen is not senior citizen")
-
-#Python program to print the largest number from two numbers.
-
-a = 215
-b = 45
-
-if a>b:
-    print("a is the largest number")
-else:
-    print("b is the largest number")
\ No newline at end of file
Index: Subramanyam/sum of prime number.py
===================================================================
diff --git a/Subramanyam/sum of prime number.py b/Subramanyam/sum of prime number.py
deleted file mode 100644
--- a/Subramanyam/sum of prime number.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,24 +0,0 @@
-num = int(input("Enter a number up to which you want to find sum: "))
-total = 0
-
-for i in range(1,num+1):
-    if i%2==0:
-        continue
-    total += i
-    print(i,end=" ")
-
-
-
-print(i,"Total: ",total)
-'''
-num = int(input("Enter a number up to which you want to find sum: "))
-total = 0
-
-for i in range(1, num + 1):
-    if i % 2 != 0:
-        continue
-    total += i
-    print(i, end=" ")
-
-print(i, "Total1: ", total)
-'''
\ No newline at end of file
Index: Sunita/New Text Document.txt
===================================================================
diff --git a/Sunita/New Text Document.txt b/Sunita/New Text Document.txt
deleted file mode 100644
--- a/Sunita/New Text Document.txt	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,1 +0,0 @@
-print("Hello world")
\ No newline at end of file
Index: Sangeetha/Python_programming/Python_Variable/pythonvaribles.py
===================================================================
diff --git a/Sangeetha/Python_programming/Python_Variable/pythonvaribles.py b/Sangeetha/Python_programming/Python_Variable/pythonvaribles.py
deleted file mode 100644
--- a/Sangeetha/Python_programming/Python_Variable/pythonvaribles.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,36 +0,0 @@
-a = 1
-b = 2
-print(a + b)
-
-#################
-a = 10
-b = 20
-print(a - b)
-
-#########
-a = 15
-b = 25
-print(a * b)
-
-##########
-str1 = "SQATOOLS"
-print("SQATOOLS" * 5)
-
-##############
-str1 = "sangeetha"
-print("sangeetha" * 5)
-
-##################
-a = 40
-b = 50
-c = 30
-print("Average: ",(a+b+c)/3)
-
-#############################
-num1 = 9
-print("square of 9 :", 9**2)
-print("cube of 9 :", 9**3)
-
-
-
-
Index: Sushmita/pythonprogram/If_else condition/Pass.py
===================================================================
diff --git a/Sushmita/pythonprogram/If_else condition/Pass.py b/Sushmita/pythonprogram/If_else condition/Pass.py
deleted file mode 100644
--- a/Sushmita/pythonprogram/If_else condition/Pass.py	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
+++ /dev/null	(revision 6780ca5441024b9c1e823e94800dd1462a11af7b)
@@ -1,5 +0,0 @@
-marks =int(input('Enter marks :'))
-if marks >= 35 :
-    print("Pass")
-else :
-    print("Fail")
Index: archana/first_python_file.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/archana/first_python_file.py b/archana/first_python_file.py
new file mode 100644
--- /dev/null	(date 1716915046910)
+++ b/archana/first_python_file.py	(date 1716915046910)
@@ -0,0 +1,1 @@
+print("hello world")
\ No newline at end of file
diff --git a/archana/if else programs b/archana/if else programs
new file mode 100644
diff --git a/Neha/New Text Document.txt b/archana/Basic Python
rename from Neha/New Text Document.txt
rename to archana/Basic Python
